package.json

```diff
@@ -11,7 +11,8 @@
     "build:all": "nx run-many --all --target=build",
     "lint:all": "nx run-many --all --target=lint",
     "e2e:all": "nx run-many --all --target=e2e",
-    "vegacapsule": "vegacapsule network bootstrap --config-path=../frontend-monorepo/vegacapsule/config.hcl"
+    "vegacapsule": "vegacapsule network bootstrap --config-path=../frontend-monorepo/vegacapsule/config.hcl",
+    "release": "git checkout develop ; git pull ; node scripts/make-release.js"
```

Comment:
mainnet shoudn't be released from master?

Reply:
true, but the tags are on develop anyway - added "mainnet" option to the script.